openid-4-verifiable-presentations-1_0.md

```diff
@@ -1817,13 +1827,18 @@ precautions against leaking information about the claim value when processing
 In both cases listed here, it needs to be considered that returning an error
 response can also leak information about the processing outcome of `values`.
 
-# Privacy Considerations
+## User Consent and Choice {#user_consent_and_choice}
 
-Many privacy considerations are specific to the credential format and associated proof type used in any particular presentation.
-This section focuses on privacy considerations that are specific to the presentation protocol with some treatment also
-given to considerations that apply across some common credential formats.
+* Wallets should obtain explicit, informed consent from the End-User before releasing any Verifiable Credential or Presentation to a Verifier.
+* If an error prevents the Wallet from honoring a request (e.g., missing credentials or mismatched claim values), the Wallet should inform the user in a privacy-preserving way.
+
+## Purpose Legitimacy and Specification {#purpose_legitimacy_and_specification}
+
+* The Verifier ensures that the purposes are sufficiently specific and are communicated before the time the information is collected. For example, it is shown before the presentation request is sent or is sent in the request to the wallet. In the latter case, the Wallet communicates them with the user before the user can make their choice. 
```

Comment:
nit suggestion: " information collection purpose is sufficiently specific and communicated before collection.
For example, the purpose is shown beforehand or included in the presentation request to the wallet. In the latter case, the wallet informs the user of the purpose before the user makes a choice"

Reply:
```suggestion
* The Verifier ensures that information collection purpose is sufficiently specific and communicated before collection.
For example, the purpose is shown beforehand or included in the presentation request to the wallet. In the latter case, the wallet informs the user of the purpose before the user makes a choice .
```